packages/eth-sdk/src/EthersClient/index.js

```diff
@@ -17,11 +18,11 @@ const { fromBech32 } = require('@harmony-js/crypto')
 
 export default class EthersClient {
   constructor(option) {
-    const { networkId = '', chainId, url } = option
+    const { networkId = '', chainId, url, group = '' } = option
     this.networkId = networkId
     this.chainId = chainId
 
-    if (window.ethereum) {
+    if (window.ethereum && networkId !== 'custom' && group !== 'others') {
```

Comment:
for better readable:
```suggestion
    const needConnectToMetamask = window.ethereum && networkId !== 'custom' && group !== 'others'
    if (needConnectToMetamask) {
```

Reply:
fine